otfmi/otfmi.py

```diff
@@ -541,6 +553,47 @@ def _set_outputs(self, outputs=None):
             outputs = self.outputs_fmu
         self.setOutputDescription(outputs)
 
+    def _set_final_time(self, final_time):
+        """Extract final time from keywords if exists.
+
+        Parameters
+        ----------
+        final_time: float (must be >= 0).
+
+        """
+        if final_time is not None:
+            self.final_time = final_time
+        else:
+            self.final_time = self.model.get_default_experiment_stop_time()
+
+    def _set_start_time(self, start_time):
+        """Extract start time from keywords if exists.
+
+        Parameters
+        ----------
+        start_time: float (must be >= 0)
+        
+        """
+        if start_time is not None:
+            self.start_time = start_time
+        else:
+            self.start_time = self.model.get_default_experiment_start_time()
+
+    def _assert_mesh_pertinence(self):
+        """Raise an error if the mesh is not comprised between the start and
+        final simulation time.
+        """
+        mesh_min = self.mesh.getValues()[0]
```

Comment:
I expected something like getOutputMesh().getVertices().getMin()[0]
Funny thing is that you can specify a mesh that travels back through time

Reply:
Ok